docs/docs/modules/chains/index.mdx

```diff
@@ -17,3 +18,23 @@ Using a language model in isolation is fine for some applications, but it is oft
 LangChain provides a standard interface for chains, as well as a number of built-in chains that can be used out of the box. You can also create your own chains.
 
 <DocCardList />
+
+## Advanced
+
+To implement your own custom chain you can subclass `BaseChain` and implement the following methods:
+
+import SubclassInterface from "@examples/chains/advanced_subclass.ts";
+
+<CodeBlock language="typescript">{SubclassInterface}</CodeBlock>
+
+### Subclassing `BaseChain`
+
+The `_call` method is the main method of a chain. It takes a record of inputs and returns a record of outputs. The inputs received should conform the `inputKeys` array, and the outputs returned should conform to the `outputKeys` array.
```

Comment:
instead of "main method of a chain," how about "main method custom chains must implement"

Reply:
Agreed!